flink-streaming-java/src/test/java/org/apache/flink/streaming/api/StreamExecutionEnvironmentTest.java

```diff
@@ -269,6 +271,27 @@ public void testGetStreamGraph() {
 		}
 	}
 
+	@Test
+	public void testDefaultJobName() {
+		StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+
+		env.fromCollection(new DummySplittableIterator<>(), BasicTypeInfo.INT_TYPE_INFO).addSink(new DiscardingSink<>());
```

Comment:
nit: simplify the test here and below to just use `env.fromElements(1, 2, 3).print()`